client/src/components/Instructions/index.js

```diff
@@ -92,6 +92,100 @@ export default class Instructions extends Component {
     );
   }
 
+  renderGaslessCounterSetup() {
+    const addressDefault = this.props.ganacheAccounts.length > 2 ? this.props.ganacheAccounts[2] : '<ADDRESS>';
+    return (
+      <div className={styles.instructions}>
+        <h2> Build your first app with ZepKit </h2>
+        <div className={styles.step}>
+          <div className={styles.instruction}>
+            1. Ensure Docker is installed on your computer.
+          </div>
+          <div className={styles.code}>
+            <code>
+              <a href="https://docs.docker.com/install/">https://docs.docker.com/install/</a>
+            </code>
+          </div>
+        </div>
+        <div className={styles.step}>
+          <div className={styles.instruction}>
+            2. In another terminal, run the gas relay docker container with ganache, wait a few seconds for it to sping up (run inside your zepkit folder).
+          </div>
+          <div className={styles.code}>
+            <code>
+              npm explore tabookey-gasless npm run gsn-dock-relay-ganache
```

Comment:
The first time I went through the tutorial, I had run this command and gone ahead with the others (since it took several minutes to complete, until the docker image was downloaded). And since I had a ganache running from the previous tutorial, most commands just worked (including the truffle migrate). 

Would it be possible to validate that the user is indeed connecting to the gsn ganache, maybe by checking that the relayer hub is deployed to the current network, and alert the user if they are not? I think this would help prevent several issues.

Reply:
I will def look into this, as others will probably run into this same issue.

Reply:
What do you think of just adding a step to kill any processes running at port 8545 before running the docker container. ie 
`npx kill-port 8545 & npm explore tabookey-gasless npm run gsn-dock-relay-ganache`

Reply:
Seems like an easy fix, I like it!